loops.js

```diff
@@ -32,7 +39,14 @@
 
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over. Google daarom eens naar 'mdn webdocs remainder operator'.
 // ==========================================
-
+for (let i = 0; i<=15; i++) {
+      if (i % 2 == 0){
+        console.log(i + " is even")
+    }
+    else{
+        console.log(i +" is oneven")
```

Comment:
je spatie-gebruik is niet altijd even consistent. Ik heb het er niet overal bijgezet.